Make Perltk build with xft support
see bug #740207 for details
patch from Roderich Schupp
--- a/myConfig
+++ b/myConfig
@@ -147,7 +147,12 @@
 {
  foreach (map { "$_/freetype2" } @_)
   {
-   if (-d $_ && -d "$_/freetype" && -r "$_/freetype/freetype.h")
+   if (-r "$_/freetype.h") # location in Debian (since jessie)
+    {
+     print "Using -I$_ to find $_/freetype/freetype.h\n";
+     return "-I$_";
+    }
+   if (-r "$_/freetype/freetype.h") # location in FreeBSD (up to version 10.0) and Debian (up to wheezy)
     {
      print "Using -I$_ to find $_/freetype/freetype.h\n";
      return "-I$_";
